plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcWriteConfig.java

```diff
@@ -21,7 +21,7 @@
 
 public class JdbcWriteConfig
 {
-    static final int MAX_ALLOWED_WRITE_BATCH_SIZE = 1_000_000;
+    public static final int MAX_ALLOWED_WRITE_BATCH_SIZE = 10_000_000;
```

Comment:
after some experience over time I notice that this isn't as useful as you'd expect.

note that batch size defines the ceiling. for table writer task to create such large batches the input splits must include this many rows which is very very unlikely in practice due to limited size of splits/pages sent to table writer tasks.

Reply:
for an ideal system once batch size is increased we should somehow signal to the engine the batch size as well so that the table writer gets sufficiently large pages. e.g. even with batch-size = 100k we can see just 10k rows per INSERT being sent.

Reply:
But we don't need need synchronize pages with batches. If there is not much data there (not enough pages) the latency of batches inserted could be perhaps lower. But if there is huge amount of data and plenty then this change was about increasing the throughput of insert.

Reply:
I agree with that however I think we can improve this further since in practice it's not always guaranteed that you'll get enough pages timely in order to be able to fill larger batches all the time.

Let's continue offline and decide the next steps if any.